.github/workflows/digest-bot.yaml

```
@@ -49,9 +49,9 @@ jobs:
         shell: bash
         run: |
           git diff --stat
-          echo "create_pr_update=false" >> $GITHUB_OUTPUT
-          if [[ $(git diff --stat) != '' ]]; then
-            echo "create_pr_update=true" >> $GITHUB_OUTPUT
+          echo "create_pr_update=false" >> "$GITHUB_OUTPUT"
+          if [[ "$(git diff --stat)" != '' ]]; then
+            echo "create_pr_update=true" >> "$GITHUB_OUTPUT"
```

```suggestion
            echo "create_pr_update=true" >> "${GITHUB_OUTPUT}"
```

sometimes it complains as well